modules/openapi-generator/src/main/resources/rust-axum/header.mustache

```diff
@@ -178,3 +178,39 @@ impl TryFrom<IntoHeaderValue<DateTime<Utc>>> for HeaderValue {
         }
     }
 }
+
+{{#renderUuidConversionImpl}}
```

Comment:
Could you please run integration test.

```
./bin/generate-samples.sh ./bin/configs/manual/*.yaml
 mvn integration-test -f samples/server/petstore/rust-axum/pom.xml
```

You might need to create new api with uuid in the header if those specs do not have

Reply:
Okay, done that. Should have done it in the first place.